nss-day-cohort-30/bangazon-site-plucky-pikes
# created shopping cart

# Description
When the user clicks 'View Cart' in the header menu, they are taken to a page that shows their open orders, including a list of products in the cart

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
1. `git checkout pf-shopping-cart`
1. run program...
1. verify...


# Checklist:
